api_tests/lib/parity_instance.ts

```diff
@@ -53,6 +53,6 @@ export class ParityInstance implements LedgerInstance {
     }
 
     public stop() {
-        this.process.kill("SIGINT");
+        this.process.kill("SIGKILL");
```

Comment:
```suggestion
        this.process.kill("SIGTERM");
```
`SIGKILL` is not really healthy, did you try `SIGTERM`?

Reply:
Unfortunately `SIGTERM` does not kill it properly. Not sure why partiy is that difficult to kill, the only one that does the job for me is `SIGKILL` 😐
I would like to investigate further, but that might be a rabbit-hole...

Reply:
I'd been to find a different strategy then as it only impacts you. Let's chat tomorrow